"list.remove(x) not in list error when in list i am randomly picking a value from a list and then removing it from the original list. i am doing this within a function and the list is out of the function. i print the list and it is in the list but, it still says it isn't. ..import random.cc = [""t:g"",""n:f""].ccr = cc.random.shuffle(ccr).def get(ccran):.   print(""cc: "", cc).   if len(ccr) &gt; 0:.       card = ccran[0].       del ccran[0].   else:.       random.shuffle(cc).       ccran = cc.       card = ccran[0].       del ccran[0].   act = card[0:2].   if act == ""t:"":.       print(""t"").       #below gives a value error..       cc.remove(""t:g"").get(ccr)...what am i doing wrong?"
